<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvToolBinarize</class>
 <widget class="QWidget" name="vvToolBinarize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binarize image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2">
        <widget class="vvIntensityValueSlider" name="mThresholdSlider2" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Foreground (FG) is greater than : </string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="vvIntensityValueSlider" name="mThresholdSlider1" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="mRadioButtonLowerThan">
         <property name="text">
          <string>and lower than</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Resulting binary image:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="mCheckBoxUseFG">
         <property name="text">
          <string>Use FG</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="vvIntensityValueSlider" name="mFGSlider" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="mCheckBoxUseBG">
         <property name="text">
          <string>Use BG</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="vvIntensityValueSlider" name="mBGSlider" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mCheckBoxInteractiveDisplay">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Interactive (unchek to disable on the fly contours drawing)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vvIntensityValueSlider</class>
   <extends>QWidget</extends>
   <header>vvIntensityValueSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mRadioButtonLowerThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>mThresholdSlider2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCheckBoxUseBG</sender>
   <signal>toggled(bool)</signal>
   <receiver>mBGSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCheckBoxUseFG</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFGSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
